Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -0,0 +1,19 @@
+module RuboCop
+  module Cop
+    class FormulaCop < Cop
+      @registry = Cop.registry
+
+      def file_inside_formula_root?
+        return false unless processed_source.path =~ %r{/Taps/homebrew}
+        true
+      end
+
+      def on_class(node)
```

Comment:
Not sure what `node` is here; could you explain so we can figure out a better name?